build/build.sh

```diff
@@ -17,19 +16,31 @@ echo "ce-build-revision:${REVISION}"
 echo "ce-build-output:${OUTPUT}"
 
 PREFIX_DIR=/opt/compiler-explorer/vala-${VERSION}
+mkdir -p "${PREFIX_DIR}"
+
+# "install" libraries and development headers required by the compiler to the prefix
+apt-get update && apt-get download libglib2.0-0 libglib2.0-dev libpcre3 libpcre3-dev
```

Comment:
Can we put these things in the docker file or do you think it would change from version to version somehow?

Reply:
There's a possibility of these changing in Ubuntu if there are minor patches for security fixes

I think including the .deb files directly just adds complexity to this because it would need work if the underlying Ubuntu version of the docker container got changed. This should just continue to work going forwards

Reply:
Understood now: thanks. I had misread :)